From 4cc4facaa35c42871ead81e4ed6d69bdc8c16e88 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@lucaweiss.eu>
Date: Mon, 12 Oct 2026 09:51:16 +0200
Subject: [PATCH] hwdb: Samsung Galaxy Book2 Go keyboard is not a graphics
 tablet

The internal keyboard of the Samsung Galaxy Book2 Go is incorrectly
tagged as a graphics tablet, breaking functionality.
---
 hwdb.d/60-input-id.hwdb | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hwdb.d/60-input-id.hwdb b/hwdb.d/60-input-id.hwdb
index 4dbadb27c42..a5ef4804197 100644
--- a/hwdb.d/60-input-id.hwdb
+++ b/hwdb.d/60-input-id.hwdb
@@ -144,3 +144,8 @@ id-input:modalias:input:b0003v11FFpA701*
 id-input:usb:v1915p1025:name:*Consumer Control*:*
  ID_INPUT_TABLET=0
  ID_INPUT_TABLET_PAD=0
+
+# Samsung Galaxy Book2 Go internal keyboard
+id-input:modalias:input:b0018v0CF2p9028e0100-e0,1,2,3,4*
+ ID_INPUT_TABLET=
+ ID_INPUT_TABLET_PAD=
